### DISCUSSION TITLE
New user: Cannot choose dataprovider (drop down is empty)

### DISCUSSION THREAD

**Discussion by einarpersson** [Budibase/budibase#9439] [Help] [answered]
Hi!

I am just trying out Budibase but I cannot figure out how to populate a simple Table.

I have added a datasource from a Google sheet, but it does not show up in the Design view.

Do I have to do something else? 

As a new user, I am lost.

![image](https://user-images.githubusercontent.com/6043691/214515331-cf2ca42b-a7e0-478a-abd2-3fc8af4b5610.png)

![image](https://user-images.githubusercontent.com/6043691/214515724-fe25f6d4-2a0f-4ef2-b897-dbd2df84b855.png)

**Comment by einarpersson** [comment, marked as answer]
Ah nevermind, I figured it out.

My confusion was that I thought that **Data Source** and **Data Provider** was the same thing.

1. Add a Data Source (eg. Google Sheet) under the Data tab
2. In the Design tab, create a Data Provider (and connect it to the Data Source in the previous step)
3. In the Design tab, create a Table (and connect it to the Data Provider)

**Reply by melohagan** [reply]
To be fair it's not super intuitive the difference between them!

I'll leave the docs page here too for reference: https://docs.budibase.com/docs/data-provider
